### DISCUSSION TITLE
About history tab in Email.js dashboard.

### DISCUSSION THREAD

**Discussion by wkdtjrrms0** [emailjs-com/help#136] [Q&A] [answered]
Hello, I am using the Email.js service well.
I'm contacting you because I have an inquiry.
In the history tab of the Email.js dashboard site(https://dashboard.emailjs.com/admin/history),
it seems that only recent mails are viewed, not all mails that have been sent so far.
Can't I look up all the mails sent in the past?

Thank you. Have a nice day 😊
![image](https://github.com/user-attachments/assets/f3eb673f-bcef-4957-9b2d-9f2ffc181144)
![image](https://github.com/user-attachments/assets/80015bc1-c280-4bde-8d09-befb1c09b523)

**Comment by xr0master** [comment, marked as answer]
Hello.
Data retention depends on your plan. For the Free plan is 7 days.
Feel free to let me know if you have more questions.
